ISSUE-MANAGEMENT.md

```diff
@@ -6,10 +6,11 @@ You can report bugs and feature requests to the Sail Operator team opening a new
 Please refer to the [Security Policy](https://github.com/istio-ecosystem/sail-operator/security/policy) for more information on how to report security issues. Please do not report security issues in the public GitHub repository.
 
 ## Community Support
-If you have questions or need help, you can reach out to the Sail Operator community in the [Istio Slack](https://istio.slack.com/). You can join the Istio Slack workspace by visiting the [Istio Slack invitation page](https://slack.istio.io/). Once you join the Istio Slack workspace, you can find the `#sail-operator` channel. Also, you can start a discussion in the [Sail Operator Discussion] (https://github.com/istio-ecosystem/sail-operator/discussions)
+If you have questions or need help, you can reach out to the Sail Operator community in the [Istio Slack](https://istio.slack.com/). You can join the Istio Slack workspace by visiting the [Istio Slack invitation page](https://slack.istio.io/). Once you join the Istio Slack workspace, you can find the `#sail-operator` channel. Also, you can start a discussion in the [Sail Operator Discussion](https://github.com/istio-ecosystem/sail-operator/discussions)
 
 ## Contributing
-If you want to contribute to the Sail Operator project, please refer to the [CONTRIBUTING-SAIL-PROJECT.md](https://github.com/istio-ecosystem/sail-operator/blob/main/CONTRIBUTING-SAIL-PROJECT.md) file for more information on how to contribute to the project. The file [CONTRIBUTING.md](https://github.com/istio-ecosystem/sail-operator/blob/main/CONTRIBUTING.md) contains all the information you need to get started with contributing Istio.
+If you want to contribute to the Sail Operator project, please refer to the [CONTRIBUTING.md](https://github.com/istio-ecosystem/sail-operator/blob/main/CONTRIBUTING.md) file for more information on how to contribute to the project.
```

Comment:
I suggest to reference the files located within the repo - locally and not by providing full path.